src/main/java/filter/expression/Qualifier.java

```diff
@@ -97,6 +97,16 @@ public static FilterExpression replaceMilestoneAliases(IModel model, FilterExpre
                 .sorted((a, b) -> getMilestoneDueDateComparator().compare(a, b))
                 .collect(Collectors.toList());
 
+        List<TurboMilestone> openMilestones = model.getMilestones().stream()
+                .filter(ms -> ms.isOpen())
+                .collect(Collectors.toList());
+
+        if (openMilestones.size() == 1 && !openMilestones.get(0).getDueDate().isPresent()) {
+            //if there is only one open issue in the repo and it doesn't have a due date, add it
+            //to the list of all milestones
+            allMilestones.addAll(openMilestones);
+        }
```

Comment:
Do we really need this? Besides, this code doesn't look like it belongs in this method. i.e. the method looks like it's doing more than one thing.